e2e/pages/adf/contentServicesPage.ts

```diff
@@ -349,23 +348,23 @@ export class ContentServicesPage {
         return deferred.promise;
     }
 
-    async checkListIsSortedByNameColumn(sortOrder) {
+    async checkListIsSortedByNameColumn(sortOrder: string) {
         return await this.contentList.dataTablePage().checkListIsSorted(sortOrder, this.columns.name);
     }
 
-    async checkListIsSortedByCreatedColumn(sortOrder) {
+    async checkListIsSortedByCreatedColumn(sortOrder: string) {
         return await this.contentList.dataTablePage().checkListIsSorted(sortOrder, this.columns.created);
     }
 
-    async checkListIsSortedByAuthorColumn(sortOrder) {
+    async checkListIsSortedByAuthorColumn(sortOrder: string) {
         return await this.contentList.dataTablePage().checkListIsSorted(sortOrder, this.columns.createdBy);
     }
 
-    async checkListIsSortedBySizeColumn(sortOrder) {
+    async checkListIsSortedBySizeColumn(sortOrder: string) {
         return await this.contentList.dataTablePage().checkListIsSorted(sortOrder, this.columns.size);
     }
 
-    sortAndCheckListIsOrderedByAuthor(sortOrder) {
+    sortAndCheckListIsOrderedByAuthor(sortOrder: string) {
```

Comment:
There are some methods that use async on this page and some that don't. it needs to be consistent across.